releasenotes/notes/deprecate-custom-basis-gates-transpile-e4b5893377f23acb.yaml

```diff
@@ -0,0 +1,18 @@
+---
+deprecations_transpiler:
+  - |
+    Providing custom gates through the ``basis_gates`` argument is deprecated 
+    for both :func:`.transpile` and :func:`generate_preset_pass_manager`, this functionality
```

Comment:
The custom `basis_gates` deprecation warning does not propagate through the `transpile` function when calling it with custom basis gates. This is because of the `warnings.simplefilter` filter here: https://github.com/Qiskit/qiskit/blob/2efb59ce4cd3c2d8721390648cdca6d2fd41f792/qiskit/compiler/transpiler.py#L398 
which is too general for this warning. It should be refined with `warnings.filterwarnings` and a regex message.